tests/behat/features/preview-navigation.feature

```diff
@@ -1,26 +1,26 @@
 Feature: Preview navigation
-  As a CMS user
-  I can navigate a subsite in the preview pane
-  In order to preview my content
+    As a CMS user
+    I can navigate a subsite in the preview pane
+    In order to preview my content
 
-  Background:
-    Given a "subsite" "My subsite"
-    And a "page" "My page" with "URLSegment"="my-page", "Content"="My page content <a name='aname'>aname</a><a href='other-page'>ahref</a>" and "Subsite"="=>Subsite.My subsite"
-    And a "page" "Other page" with "URLSegment"="other-page", "Content"="Other page content <form action='my-page'><input type='submit' value='Submit my form'></form>" and "Subsite"="=>Subsite.My subsite"
-    Given a "member" "Joe" belonging to "Admin Group" with "Email"="joe@test.com" and "Password"="Password1"
-    And the "group" "Admin Group" has permissions "Full administrative rights"
-    And I log in with "joe@test.com" and "Password1"
+    Background:
+        Given a "subsite" "My subsite"
```

Comment:
Hmmm, I think 2 spaces for indentation was good for behat, can you revert this file?